hub/config/src/main/java/uk/gov/ida/hub/config/data/S3ConfigSource.java

```diff
@@ -37,12 +38,14 @@ public S3ConfigSource(ConfigConfiguration configConfiguration, AmazonS3 s3Client
         CacheLoader<String, RemoteConfigCollection> loader;
         loader = new CacheLoader<String, RemoteConfigCollection>() {
             private RemoteConfigCollection currentConfig;
-            private Date lastModified;
 
             @Override
             public RemoteConfigCollection load(String key) {
                 S3Object s3Object = getS3Object(key);
-                if (this.lastModified.before(s3Object.getObjectMetadata().getLastModified())) {
+                if(lastModified == null) {
+                    updateObjectData(getS3Object(key));
```

Comment:
I think now might be the time to extract the s3 client code into its own class as there's a lot going on here now.